shell/server/pre-meteor.js

```diff
@@ -129,6 +129,13 @@ Meteor.startup(function () {
   });
 });
 
+var errorTxtMapping = {};
+errorTxtMapping[Dns.NOTFOUND] = "\nIf you were trying to connect this address to a Sandstorm app hosted at this server,\n" +
+    "you either have not set your DNS TXT records correctly or the DNS cache has not\n" +
+    "updated yet (may take a while).\n";
+errorTxtMapping[Dns.TIMEOUT] = "\nThe DNS query has timed out, which may be a sign of poorly configured DNS on the server. \n";
+errorTxtMapping[Dns.CONNREFUSED] = "\nThe DNS server refused the connection, which means either your DNS server is down/unreachable, or the server has misconfigured their DNS. \n";
```

Comment:
Stick a newline somewhere in here to match the wrapping of the other text. (Also, break the string to wrap the code line.)